CS483/CS483/Kartaclysm/StateMachine/Gameplay/StateRacing.cpp

```diff
@@ -226,7 +226,7 @@ void Kartaclysm::StateRacing::CreateHUDForRacer(const std::string& p_strGuid)
 
 void Kartaclysm::StateRacing::Suspend(const int p_iNewState)
 {
-	m_bSuspended = true;
+	m_bSuspended = (p_iNewState != STATE_COUNTDOWN);
```

Comment:
Okay, that makes sense. Just seems a little hacky since the Suspend call isn't actually Suspending the state under the specific circumstance, but it works.